DigitalLearningSolutions.Web/Controllers/LearningSolutions/SupportController.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.LearningSolutions
+{
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.LearningSolutions;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class SupportController : Controller

```

Comment:
I just realised that this should be available for only CentreAdmins and Frameworks developers for now - do you mind updating the JIRA ticket too? 🙏